### ISSUE TITLE
NAN on the evaluation

### ISSUE THREAD

**Issue by arpanpoudel** [GiovanniPasq/DA-Faster-RCNN#19]
Hi,
Thank you for the awesome documentation.

However, I have one issue. I am getting NAN during evaluation during training. Could you provide insight ( I am doing a single object detection)
<img width="678" alt="image" src="https://github.com/user-attachments/assets/784404b2-1ae0-443b-8d6f-559c828d3bcc">

**Comment by GiovanniPasq** [comment]
Your model has a mAP of 0.075 for the 'Broiler' class. It seems there may be issues with the annotations in your training set. Additionally, please note that a test set of only 10 images is quite small and may not provide reliable results. NaN results are due to values close to zero, whose division gives NaN when the metric is computed.

**Comment by arpanpoudel** [comment]
> Your model has a mAP of 0.075 for the 'Broiler' class. It seems there may be issues with the annotations in your training set. Additionally, please note that a test set of only 10 images is quite small and may not provide reliable results. NaN results are due to values close to zero, whose division gives NaN when the metric is computed.

Thank you for insights. I found that the bounding box size in the test set doesn't exist for mAPs(32x32) and mAP(forgot size) and because of that, I am getting the NaN error.
